### PR TITLE
TS cleanup

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary

TS cleanup

- Enable back disabled tests
- Update reasons for the test which should be disabled

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [x] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: pom.xml
Comment (rsvoboda) [inline pom.xml:30]: Shouldn't this be removed? I'm not sure if we have any camel related test coverage (anymore).
And the commit is not enabling any.

**Comment by rsvoboda** [comment]
I'm still counting on @michalvavrik to do proper review, I just scanned the changes ...

**Comment by michalvavrik** [comment]
> I'm still counting on @michalvavrik to do proper review, I just scanned the changes ...

damn, ok, will do

**Comment by michalvavrik** [comment]
I'll wait with review till I can see CI passing if you don't mind.

File: pom.xml
Comment (fedinskiy) [reply pom.xml:30]: Since @michalvavrik was the last person to change this in https://github.com/quarkus-qe/quarkus-test-suite/pull/1060 , this is probably a question for him

Comment (michalvavrik) [reply pom.xml:30]: I'd definitely remove it. I think removed Kamelet module and didn't do it. Let's drop it.

Comment (fedinskiy) [reply pom.xml:30]: Removed

**Comment by michalvavrik** [comment]
Windows failures are consequence of enabling these tests. Please debug it.

File: build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/CliProdModeNativeIT.java
Comment (michalvavrik) [inline build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/CliProdModeNativeIT.java:25]: jeez, good catch

File: http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java
Comment (michalvavrik) [inline http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java:18]: Yeah, it is fixed for a long time, but I never got to fix it on our side. If you really want to drop this flag, we need green FIPS run. I have vague memory it didn't work when I dropped it and it required more work, but it's a _long_ time ago. Things could have changed. Please run it on FIPS and confirm back.

Comment (michalvavrik) [reply http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java:18]: BTW it is nice to have; we need to know that Quarkus side works on FIPS enabled env, not the integrations. But there is definitely work on Keycloak / mTLS Keycloak tests that deservers to be done when we have a time.

File: messaging/infinispan-grpc-kafka/src/test/java/io/quarkus/ts/messaging/infinispan/grpc/kafka/InfinispanKafkaSaslSslIT.java
Comment (michalvavrik) [inline messaging/infinispan-grpc-kafka/src/test/java/io/quarkus/ts/messaging/infinispan/grpc/kafka/InfinispanKafkaSaslSslIT.java:30]:
```suggestion
            .onPreStart((action) -> CertUtils.prepareCerts());
```

**Review by michalvavrik — COMMENTED** [review]
Finished review. Changes LGTM, I left few comments. Only thing I am actually concerned is KC in FIPS. If you verify it in OpenStack for most of the tests, I am fine. Make sure you run OIDC CLient mTLS one as well.

Nice work.

**Comment by michalvavrik** [comment]
Also, we will need green OCP run, but let's wait till GH CI is green. I'll merge it once everything is green later this week.

**Comment by fedinskiy** [comment]
run tests

File: http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java
Comment (michalvavrik) [reply http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java:18]: If you can check `io.quarkus.ts.security.oidcclient.mtls.Pkcs12OidcMtlsIT` I'd appreciate it. The difference is in secured communication.

Anyway, as  you already checked the HTTP module, it means that we can remove it at least somewhere. Thanks

Comment (fedinskiy) [reply http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java:18]: These are the results for `mvn clean verify -pl security/oidc-client-mutual-tls/`:
```
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 27.21 s -- in io.quarkus.ts.security.oidcclient.mtls.Pkcs12OidcMtlsIT
```

Comment (michalvavrik) [reply http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java:18]: Thank you

**Review by michalvavrik — APPROVED** [review]
Approved provided the rest of CI goes as well as was is already finished (I'll check). Thanks for this PR.

**Pull request merged**
